packages/dm-core-plugins/src/form/templates/ArrayPrimitiveDataGridTemplate.tsx

```diff
@@ -43,7 +43,7 @@ export const ArrayPrimitiveDatagridTemplate = (
   return (
     <Fieldset>
       <Legend data-testid='primitive-array-legend'>
-        <Typography bold={true}>{getDisplayLabel(attribute)}</Typography>
+        <Typography bold={true}>{getDisplayLabel(attribute, true)}</Typography>
```

Comment:
May I ask why so many fields are set to default hide the optional label? I'm sure there's a reason, just a bit confusing 🙂

Reply:
I say many, but it's not a lot! I just feel like these CAN be optional, no?

Reply:
Hmm I think you cannot delete completely a primitive array? 
You simply delete all elements? Perhaps I am mistaken.

Reply:
Huuuuh, I don't know! 😅 You probably should be able to remove it if it is optional (even tough it doesn't look like you can delete after instantiating right now)
Here the Numbers only field is optional, it should probably say so? I mean, if we're marking things that are optional with optional, we should probably tag all things that are optional with optional otherwise I feel like it's inconsistent and confusing for the user

![Skjermbilde 2024-01-10 kl  14 45 07](https://github.com/equinor/dm-core-packages/assets/25383299/e289904f-ce38-42a6-af7e-8bc9695cfa62)

Reply:
Fair point, I'll change it.